README.md

```diff
@@ -42,6 +42,7 @@ missing please open an issue. If you are interested in joining check out
 - [@rvagg](https://github.com/rvagg) - Rod Vagg
 - [@Starefossen](https://github.com/Starefossen) - Hans Kristian Flaatten
 - [@Trott](https://github.com/Trott) - Rich Trott
+- [@lucalanziani](https://github.com/lucalanziani) - Luca Lanziani
```

Comment:
GitHub usernames are sorted alphabetically :)

Reply:
ah, I've checked if the order was on first names, didn't think of checking the github usernames, fixing that.